src/composables/useTresContextProvider/index.ts

```diff
@@ -185,6 +193,31 @@ export function useTresContextProvider({
     root: ctx,
   }
 
+  // The loop
+
+  ctx.loop.register(() => {
```

Comment:
**question:** Can/Should we make sure that this is resored once a user replaced the rendering with a custom callback and removed it afterwards via `off`?